src/data/enums/roles.enums.ts

```diff
@@ -0,0 +1,5 @@
+export enum UserRoles {
+  SUPER_ADMIN = 'super_admin',
+  ADMIN = 'admin',
+  USER = 'user',
+}
```

Comment:
Please rebase from main. 

```suggestion
export enum UserRoles {
  SUPER_ADMIN = 'super_admin',
  ADMIN = 'admin',
  USER = 'user',
}
```